h/services/group_create.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+from functools import partial
```

Comment:
Nit, there should be a blank line between future imports and stdlib imports. I think our conventions are basically what [isort](https://pypi.org/project/isort/) / Black do.

Reply:
This is really helpful. I had no idea! Could something like Black (in the future) help us with these convention-level things?

Reply:
Yep, I think Black will do this when we get round to integrating it.